### PR TITLE
wallet2_api: add solo mining API

### PR DESCRIPTION
**Pull request by moneromooo-monero**

### CODE REVIEW

File: src/wallet/api/wallet_manager.cpp [hunk 1]
```diff
@@ -345,13 +345,49 @@ double WalletManagerImpl::miningHashRate() const
     cryptonote::COMMAND_RPC_MINING_STATUS::response mres;
 
     epee::net_utils::http::http_simple_client http_client;
-    if (!epee::net_utils::invoke_http_json_remote_command2(m_daemonAddress + "/getinfo", mreq, mres, http_client))
+    if (!epee::net_utils::invoke_http_json_remote_command2(m_daemonAddress + "/mining_status", mreq, mres, http_client))
```
Comment (vtnerd) [inline src/wallet/api/wallet_manager.cpp]: Was this just the wrong RPC call?

Comment (moneromooo-monero) [inline src/wallet/api/wallet_manager.cpp]: Yes, a copy/paste which I forgot to change the destination.

**Comment by fluffypony** [comment]
Plz rebase

**Comment by moneromooo-monero** [comment]
done

**Pull request merged**